pulser/tests/test_simconfig.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Pulser Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from pulser.simulation import SimConfig
+
+
+def test_init():
+    config = SimConfig()
+    str(config)
```

Comment:
You should initialize a `SimConfig` with custom values and assert that `str(config)` matches what you would expect